tools/dts/quartz64.dts

```diff
@@ -0,0 +1,2925 @@
+/*
+ * Copyright Linux Kernel Team
```

Comment:
a quick question. do we need to include a full-dts here even if most of the devices will be unused?

Reply:
It does not harm and it can be checked against the most recent changes in the Linux repo then easily. So unless there is a problem I think that is preferred over a hand made custom one.